roles/pulp3/README.md

```diff
@@ -1,14 +1,56 @@
 Pulp3
 =====
 
-Ansible role that installs Pulp 3 from PyPi and provides basic config.
+Ansible role that installs Pulp 3 from PyPi or source and provides basic config.
 
 The default administrative user for the Pulp application is: 'admin'
 
-Example Playbook
-----------------
-
-https://raw.githubusercontent.com/pulp/devel/3.0-dev/ansible/deploy-pulp3.yml
+Role Variables:
+---------------
+
+* `pulp_cache_dir`: Location of Pulp cache. Defaults to "/var/cache/pulp".
+* `pulp_config`: Defaults to '{}'. Configuration added to Pulp's Django settings.
+* `pulp_config_dir`: Directory which will contain Pulp configuration files.
+  Defaults to "/etc/pulp".
+* `pulp_default_admin_password`: Initial password for the Pulp admin. **Required**.
+* `pulp_install_dir`: Location of a virtual environment for Pulp and its Python
+  dependencies. Defaults to "/usr/local/lib/pulp".
+* `pulp_install_plugins`: A nested dictionary of plugin configuration options.
+  Defaults to "{}", which will not install any plugins.
+  * Dictionary Key: The pip installable plugin name. This is defined in each
+  plugin's* `setup.py`. **Required**.
+  * `app_label`: Used to make migrations. This is defined in the
+  [PluginAppConfig](https://github.com/pulp/pulp_file/blob/76cc979e67fde128f78f3274697a4ea78f2269ec/pulp_file/app/__init__.py#L10)
+  of each plugin. **Required**.
+  * `source_dir`: Optional. Absolute path to the plugin source code. If present,
+  plugin will be installed from source in editable mode.
+* `pulp_install_service_files`: Whether to create systemd service files for
```

Comment:
is this just for the web server or is this used by the workers also?

Reply:
This is just wsgi.